docs/blocks/feature-flags-and-experimental-interfaces.md

```diff
@@ -0,0 +1,53 @@
+We have feature gating system setup in our plugin that defines what is accessible to the public and what is not, it has three phases:
+
+*   **Core flag `WOOCOMMERCE_BLOCKS_PHASE=1`:** anything that is not hidden behind a flag, falls under this category, and it contains all of the code that runs on WooCommerce Core plugin.
+*   **Feature plugin flag** `**WOOCOMMERCE_BLOCKS_PHASE=**2`**:** anything that is behind this flag is code that is shipped to our [feature plugin](https://wordpress.org/plugins/woo-gutenberg-products-block/), the files of blocks behind this flag are also present in WooCommerce Core, just not active.
+*   **Experimental flag** `**WOOCOMMERCE_BLOCKS_PHASE**=3`: This flag contains things that we're not shipping yet, so unfinished work mostly. These features are only available in developer builds of the plugin.
+
+We also utilize the following for indicating interfaces that are experimental and thus should not be used in production builds:
+
+- `__experimental` prefix is used in any PHP actions or filter slugs and also for any JS functions.
+
+## Blocks behind flags
+
+The majority of our feature flagging is blocks, this is a list of them:
+
+### Feature plugin flag
+
+*   Cart block ([JS flag](https://github.com/woocommerce/woocommerce-gutenberg-products-block//blob/9b76ea7a1680e68cc20bfee01078e43ccfc996bd/assets/js/blocks/cart-checkout/cart/index.js#L53) | [PHP flag 1](https://github.com/woocommerce/woocommerce-gutenberg-products-block//blob/e167b2c99c68e8113b4e371fefdd6f9a356ed2e8/src/Library.php#L74-L77) | [PHP flag 2](https://github.com/woocommerce/woocommerce-gutenberg-products-block//blob/b3a9753d8b7dae18b36025d09fbff835b8365de0/src/Assets.php#L92-L95)).
```

Comment:
Not a big deal, but some links have double `//` before `blob`. I don't think it's blocking because GH ignores them but I assume the correct URL would be with a single slash.
```suggestion
*   Cart block ([JS flag](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/9b76ea7a1680e68cc20bfee01078e43ccfc996bd/assets/js/blocks/cart-checkout/cart/index.js#L53) | [PHP flag 1](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/e167b2c99c68e8113b4e371fefdd6f9a356ed2e8/src/Library.php#L74-L77) | [PHP flag 2](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/b3a9753d8b7dae18b36025d09fbff835b8365de0/src/Assets.php#L92-L95)).
```

Reply:
Another great catch!